backend/core/src/listings/listings.service.ts

```diff
@@ -147,12 +151,20 @@ export class ListingsService {
     return paginatedListings
   }
 
-  async create(listingDto: ListingCreateDto) {
+  async create(listingDto: ListingCreateDto): Promise<Listing> {
     const listing = this.listingRepository.create({
       ...listingDto,
       property: plainToClass(PropertyCreateDto, listingDto),
     })
-    const saveResult = await listing.save()
+    const saveResult: Listing = await listing.save()
+
+    // Add a job to the listings notification queue, to asynchronously send notifications about
+    // the creation of this new listing.
+    await this.listingsNotificationsQueue.add({
+      listing: saveResult,
+      updateType: ListingUpdateType.CREATE,
```

Comment:
Do you want to add the `MODIFY` case in `update` below, or save for a later PR?

Reply:
Thanks for the quick review!

On this: I think I'll wait for a later PR, since I'm not clear on the answer to "which types of modifications warrant notifications?"